src/main/java/com/matera/account/account/AccountController.java

```diff
@@ -16,6 +16,12 @@ public AccountController(AccountService accountService) {
         this.accountService = accountService;
     }
 
+    @GetMapping("/clients")
+    // List ALL clients
+    public List<Account> getAccountsByClientId() {
```

Comment:
This method name shouldn't be `getAccounts` or `getAllAccounts`? Because we are not using any ClientId